mmv1/products/accesscontextmanager/AccessLevelCondition.yaml

```diff
@@ -248,3 +249,10 @@ properties:
               description: 'CIDR block IP subnetwork specification. Must be IPv4.'
               item_type:
                 type: String
+  - name: 'accessPolicyId'
+    type: String
+    description: |
+      The name of the Access Policy this resource belongs to.
+    ignore_read: true
```

Comment:
Why is this field `immutable` and `output true` at the same time? 

Can the user set the policyId from their TF config?

This comment also applies for all the cases below.

Reply:
Thanks for the explanation. 
If `immutable` is set to true, if there's any modification to this field, another new resource will be recreated.  Is that something you'd expect?